crates/rpc/src/routes/config.rs

```diff
@@ -3,16 +3,28 @@ use std::sync::Arc;
 use ream_network_spec::networks::NetworkSpec;
 use warp::{Filter, Rejection, filters::path::end, get, log, path, reply::Reply};
 
-use crate::handlers::config::get_deposit_contract;
+use crate::handlers::config::{get_deposit_contract, get_spec};
 
+/// Creates and returns all `/config` routes.
 /// Creates and returns all `/config` routes.
 pub fn get_config_routes(
     network_spec: Arc<NetworkSpec>,
 ) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
-    path("config")
+    let deposit_network_spec = network_spec.clone();
+    let spec_network_spec = network_spec.clone();
+
+    let deposit_contract = path("config")
```

Comment:
could you make a filter like we use for passing in the database, instead of cloning it twice at the top like this?